### PR TITLE
Use Template Literals

### PR DESCRIPTION
**Pull request by ghost**
**Template literals are string templates that can include variables and expressions.** They can replace string concatenation and are often easier to read. Example: `` `Hello, ${name}!` ``

This refactoring supports converting the first line of concatenated strings into template literals or shortened strings:
* concatenated string literals will be combined, e.g. `"a" + "b"` will become `"ab"`
* concatenated expressions and variables will be embedded in templates, e.g. `"a" + x + "b"` will become `` `a${x}b` ``
* unnecesary escaping of quotes is removed when string are combined or transformed into template literals

Learn More: [Template Literals (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

### CODE REVIEW

File: packages/core/src/utils/set_immediate.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
     function setImmediate(callback) {
         // Callback can either be a function or a string
         if (typeof callback !== 'function') {
-            callback = new Function('' + callback);
+            callback = new Function(`${callback}`);
```
Comment (domoritz) [inline packages/core/src/utils/set_immediate.js]:
```suggestion
            callback = new Function(String(callback}));
```

File: packages/core/src/utils/set_immediate.js [hunk 2]
```diff
@@ -14,7 +14,7 @@
     function setImmediate(callback) {
         // Callback can either be a function or a string
         if (typeof callback !== 'function') {
-            callback = new Function(`${callback}`);
+            callback = new Function(String(callback}));
```
Comment (ankoh) [inline packages/core/src/utils/set_immediate.js]: Curly

Comment (domoritz) [reply packages/core/src/utils/set_immediate.js]: Ups

Comment (domoritz) [reply packages/core/src/utils/set_immediate.js]:
```suggestion
            callback = new Function(String(callback));
```

**Pull request merged**